app/helpers/application_cocoon_helper.rb

```diff
@@ -1,42 +1,48 @@
-  #     selection.
-  #   @option html_options [Symbol] :selector The jQuery selector to be used with +:find_using+ to
-  #     determine where to insert the new node.
-  #   @option html_options [Symbol] :find_selector The alias of +:selector+, you should use this
+  #   @option html_options [String] :find_using The jQuery traversal method to allow node selection
+  #     relative to the link. Example values: +this+, +closest+, +next+, +children+, etc.
+  #     Default: absolute selection.
+  #   @option html_options [String] :selector The jQuery selector to be used with +:find_using+ to
+  #     determine where to insert the new node. Default: the parent node.
+  #   @option html_options [String] :find_selector The alias of +:selector+, you should use this
   #     when +:find_using+ is not specified. Default: parent node.
-  #   @option html_options [Symbol] :insert_using ('before') The jQuery method that inserts the
-  #     new data. Example values: `before`, `after`, `append`, `prepend`, etc.
-  #   @param [Proc] block The block to use for displaying the link.
+  #   @option html_options [String] :insert_using ('before') The jQuery method that inserts the
+  #     new data. Example values: +before+, +after+, +append+, +prepend+, etc.
+  #   @return [String]
+  #   @example Adding items after the link
+  #     link_to_add_association(t('.add_option'), f, :options, find_selector: 'this',
+  #                                                            insert_using: 'after')
   # @overload link_to_add_association(form, options, association, html_options = nil, &block)
   #   Shows a link that will allow to dynamically add a new associated object. The display of the
   #   link can be changed in the block.
+  #
   #   @param [String] name The text to show in the link.
   #   @param [ActionView::Helpers::FormBuilder] form The form builder.
   #   @param [Symbol] association The associated objects, this should be the name of the has_many
-  #   relation.
+  #     relation.
   #   @param [Hash] html_options The html options to be passed to {#link_to}.
-  #   @option html_options [Symbol] :find_using The jQuery traversal method to allow node selection
-  #     relative to the link. Example values: `closest`, `next`, `children`, etc. Default: absolute
-  #     selection.
-  #   @option html_options [Symbol] :selector The jQuery selector to be used with +:find_using+ to
-  #     determine where to insert the new node. Default: parent node.
-  #   @option html_options [Symbol] :find_selector The alias of +:selector+, you should use this
+  #   @option html_options [String] :find_using The jQuery traversal method to allow node selection
+  #     relative to the link. Example values: +this+, +closest+, +next+, +children+, etc.
+  #     Default: absolute selection.
+  #   @option html_options [String] :selector The jQuery selector to be used with +:find_using+ to
```

Comment:
these are JavaScript values, not Ruby values, so actually symbol or string doesn't make a difference. Grepping shows strings are used more than symbols